02/app.js

```diff
@@ -9,12 +9,21 @@ class App extends React.Component {
         usersList: [],
     }
 
+    updateUsersList = evt => {
+      evt.preventDefault();
+      const usersList = this.state.usersList.map(item => item);
+      const input = evt.currentTarget.querySelector('input');
```

Comment:
Poto tworzyliśmy `ref`, aby z niego skorzystać :P Wystarczy zrobić tak, że metoda `updateUsersList` przyjmuje wartość to wprowadzenia do `state`. Natomiast całą metodą przekazujemy do `Form` przez props i tam jest ona wykorzystywana.

WYkorzystywanie `querySelector` w React nie jest dobrym pomysłem. Tutaj nie DOM ma być źródłem prawdy a komponenty - ich stan.